resources/views/widgets/widget.blade.php

```diff
@@ -0,0 +1,53 @@
+@php
+    $heading = $this->getHeading();
+    $tableHeading = $this->getTableHeading();
+    $filters = $this->getFilters();
+@endphp
+
+<x-filament::widget>
+    <x-filament::card>
+        @if ($heading || $filters)
+            <div class="flex items-center justify-between gap-8">
+                @if ($heading)
+                    <x-filament::card.heading>
+                        {{ __($heading) }}
+                    </x-filament::card.heading>
+                @endif
+
+                @if ($filters)
+                    <select
+                            wire:model="filter"
+                            @class([
+                                'text-gray-900 border-gray-300 block h-10 transition duration-75 rounded-lg shadow-sm focus:border-primary-500 focus:ring-1 focus:ring-inset focus:ring-primary-500',
+                                'dark:bg-gray-700 dark:border-gray-600 dark:text-gray-200 dark:focus:border-primary-500' => config('filament.dark_mode'),
+                            ])
+                            wire:loading.class="animate-pulse"
```

Comment:
Could we indent this back a tab, please? Everything should indent 4 spaces, and this looks like 8?